src/context/context.service.ts

```diff
@@ -78,10 +79,18 @@ export class ContextService {
     return this.title;
   }
 
+  getVersion() {
+    return this.version;
+  }
+
   setTitle(title: string): void {
     this.title = title;
   }
 
+  setVersion(version: any): void {
```

Comment:
Let's add a `context.interface.ts` with a new type Version so we get proper typing for the return of `getVersion() :Version` and the parameter passed to `setVersion(version: Version)`

Reply:
Yes i added a new interface for version with the 3 parameters. (we can add others if we want)